core/src/mindustry/entities/type/Player.java

```diff
@@ -273,6 +273,10 @@ public void setTeam(Team team){
         this.team = team;
     }
 
+    public float getItemtime(){
```

Comment:
Instead of making a getter for this, you should add a method to `Player`:
```java
public float drawBackItems(){
    drawBackItems(itemtime, isLocal);
}
```

...and call this method in the mech.

Reply:
gotcha, will update it

Reply:
implemented void drawBackItems in player and mech drawStats